lib/cookie/index.js

```diff
@@ -79,6 +79,16 @@ module.exports = {
     if (consentPresent) {
       createCookie(name, value, days, domain, secure);
     }
+    cookieConsent.subscribe(name,'add',function (name) {
+      if (cookieConsent.getConsent(name.toString())) {
```

Comment:
rename the param to be `consent`
Use the `consent` param instead of get it from getConsent

Reply:
Made the changes